packages/app/package.json

```diff
@@ -37,6 +37,7 @@
     "@urql/vue": "0.4.3",
     "@vitejs/plugin-vue": "1.2.4",
     "@vitejs/plugin-vue-jsx": "1.1.6",
+    "@vue/compiler-sfc": "^3.2.11",
```

Comment:
Just gonna make this 3.2.0 to match up with the verison of Vue. Generally you want the Vue version and compiler version to be the same, otherwise it's possible the compiler generates some code that's not compatible with the Vue version.